servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -235,6 +226,16 @@ public Completable closeAsyncGracefully() {
         }
     }
 
+    /**
+     * Retrying filter that disables any form of retry behaviour. All types of failures will not be re-attempted.
+     * @return a retrying filter that disables any form of retry behaviour. All types of failures will not be
+     * re-attempted.
+     */
+    public static RetryingHttpRequesterFilter disableAutoRetries() {
```

Comment:
IIUC, you plan to cherry-pick the new approach into 0.41. Make sure that `AutoRetryStrategyProvider#DISABLE_AUTO_RETRIES` references this new method as a replacement